modules/web-console/src/main/java/org/apache/ignite/console/config/WebConsoleConfigurationHelper.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.console.config;
+
+import java.io.File;
+import java.net.URL;
+import java.util.function.Function;
+import io.vertx.config.ConfigChange;
+import io.vertx.config.ConfigRetriever;
+import io.vertx.config.ConfigRetrieverOptions;
+import io.vertx.config.ConfigStoreOptions;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.streams.ReadStream;
+import org.apache.ignite.Ignition;
+
+/**
+ * Configuration helper.
+ */
+public class WebConsoleConfigurationHelper {
+    /**
+     * Config format.
+     */
+    public enum Format {
+        /** */
+        JSON,
+
+        /** */
+        PROPERTIES,
+
+        /** */
+        XML
+    }
+
+    /**
+     * @param vertx Vertx.
+     * @param fmt File format: JSON or properties.
+     * @param path Path to configuration file.
+     * @param beanName Optional, bean name, required only for XML configuration.
+     * @return Web Console configuration receiver.
+     */
+    public static ConfigRetriever loadConfiguration(Vertx vertx, Format fmt, String path, String beanName) {
+        if (fmt == Format.XML)
+            return new XmlConfigRetriever(path, beanName);
+
+        ConfigStoreOptions storeOpts = new ConfigStoreOptions()
+            .setType("file")
+            .setFormat(fmt.name().toLowerCase())
+            .setConfig(new JsonObject().put("path", path));
+
+        return ConfigRetriever.create(vertx, new ConfigRetrieverOptions().addStore(storeOpts));
```

Comment:
You should create one ConfigRetriever add multiple store (env, properties file, kubernetes config map)